.ahub/sam/exclude.txt

```diff
@@ -5,6 +5,37 @@
 # Eigen
 /ONE/compiler/nnc/backends/soft_backend/code_snippets/eigen.def
 
+# Frontend obsolete
+/ONE/compiler/caffe2circle
+/ONE/compiler/mio-tflite260
+
+# Frontend test codes
```

Comment:
For test codes, #8881 might be better solution :)

Reply:
Thanks, you gave me a good idea :)